src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlCommandExtended.cs

```diff
@@ -0,0 +1,203 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using static Interop.VoiceControlManager;
+using static Interop.ControlCommandExpand;
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// This Class represents the Extended Command
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class VoiceControlCommandExtended : VoiceCommand
+    {
+        private VoiceCommand _cmd;
+
+        /// <summary>
+        /// Public Constructor
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <param name="cmd">Voice Command</param>
+        public VoiceControlCommandExtended(VoiceCommand cmd)
+        {
+            _cmd = cmd;
+        }
+
+        /// <summary>
+        /// Gets or Sets command domain.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <exception cref="UnauthorizedAccessException">This exception can be due to permission denied.</exception>
+        /// <exception cref="NotSupportedException">This exception can be due to not supported.</exception>
+        public int Domain
```

Comment:
What is Domain? why is it int type?

Reply:
We will remove VoiceControlCommandExtended.cs